src/kubectl.ts

```diff
@@ -219,7 +219,6 @@ async function asLines(context: Context, command: string): Promise<Errorable<str
     const shellResult = await invokeAsync(context, command);
     if (shellResult.code === 0) {
         let lines = shellResult.stdout.split('\n');
-        lines.shift();
```

Comment:
I think removing this line will break other consumers of `asLines`, which expect the header line to have been removed.

Reply:
I suggest we should either:

* Have two functions, `asLines` and `asLinesWithHeader` (both returning string arrays)
* Have two functions, `asLines` (unparsed) and `fromLines` (parsed), the former returning a string array and the latter an object array
* Have a `keepHeaderLine` argument (requires changing all calls to specify whether they want to keep the header line)
* Return a `{ headerLine: string; resourceLines: string[] }` instead of a `string` (requires changing all calls to discard the header line, which I know could be done with an `unshift` but this strategy gives us compiler checking)

I quite like the second option as it minimises changes elsewhere in the codebase, and as the parsed `fromLines` is probably going to be handy going forward.  But not quite sure, and naming is going to be tricky...

Reply:
Sounds good will go for second option.